Erreur avec webpack :  not found: Error: Default condition should be last one

Bonjour, 
Depuis la release 4.0.1, il est impossible d'utiliser le package dans une application construite par webpack.

A la compilation, l'erreur suivante est lancée : 
Module not found: Error: Default condition should be last one

Après quelques recherches, il semble que dans la partie "exports" de package.json, "default" devrait être le dernier élément de la liste. Or dans les fichiers de release, "types" intervient après "default"

refs : 
[doc webpack](https://webpack.js.org/guides/package-exports/#conditional-syntax)
[doc node](https://nodejs.org/api/packages.html#conditional-exports)

### Comment reproduire le bug : 

initialiser un projet node avec le fichier package.json suivant : 

``` json
{
  "name": "untitled",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "build": "webpack"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "webpack": "^5.88.1",
    "webpack-cli": "^5.1.4"
  },
  "dependencies": {
    "@gouvminint/vue-dsfr": "^4.1.0"
  }
}

```

créer un fichier src/index.js avec le contenu suivant : 

``` js
import VueDsfr from '@gouvminint/vue-dsfr'
```

executer `npm run build`

### Contournement du problème : 

Utiliser la version 4.0.0 du package vue-dsfr

modifier le package.json du package dans node module pour placer les "export.default" en dernière position (non reecommandé, uniquement pour devs)

--- a/package.json
+++ b/package.json
@@ -19,12 +19,12 @@
   "exports": {
     ".": {
       "require": {
-        "default": "./dist/vue-dsfr.umd.js",
-        "types": "./types/index.d.ts"
+        "types": "./types/index.d.ts",
+        "default": "./dist/vue-dsfr.umd.js"
       },
       "import": {
-        "default": "./dist/vue-dsfr.mjs",
-        "types": "./types/index.d.ts"
+        "types": "./types/index.d.ts",
+        "default": "./dist/vue-dsfr.mjs"
       }
     },
     "./dist/vue-dsfr.css": {

